packages/framework/src/evaluate/binding.ts

```diff
@@ -92,9 +92,7 @@ export const createBindingAtom = <T = unknown>(
             ? get(screenStorageAtom)
             : undefined,
         ),
-        user: rawJsExpression.includes("ensemble.user")
-          ? get(userAtom)
-          : undefined,
+        user: get(userAtom),
```

Comment:
Can you revert this change? We don't want to create unnecessary depenencies.

Reply:
so should I remove the get method from the useEmsembleUser hook ?

Reply:
No, why would you do that? I'm just asking to revert these three lines here.

Reply:
oh, I changed this while I changed user atom from state to hook. this is for the test only and it was need to be reverted back.